From 64298fd57f632afcf3b6bf87c10887fbf74413a0 Mon Sep 17 00:00:00 2001
From: Ziyuan Xu <xzy.xu@rock-chips.com>
Date: Thu, 19 Apr 2018 11:36:07 +0800
Subject: [PATCH] arm64: dts: rockchip: assign 400KHz to i2c bus clock for
 rk3308-evb

Reduce about 8ms for led driver initialization time

before:
initcall is31fl32xx_driver_init+0x0/0x20 returned 0 after 14832 usecs

after:
initcall is31fl32xx_driver_init+0x0/0x20 returned 0 after 6270 usecs

Change-Id: I3154efa154a484ccf8f9974c8148ff9895b1f68a
Signed-off-by: Ziyuan Xu <xzy.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
index 5e34c8a081bf..9f83a2c7207e 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
@@ -272,6 +272,7 @@
 };
 
 &i2c1 {
+	clock-frequency = <400000>;
 	status = "okay";
 
 	is31fl3236: led-controller@3c {
-- 
2.35.3

